src/components/Header/Header.tsx

```diff
@@ -1,17 +1,24 @@
 import SearchIcon from '@mui/icons-material/Search';
 import { AppBar, Box, IconButton, Toolbar, Typography } from '@mui/material';
+import { useRouter } from 'next/router';
 import { useState } from 'react';
 
 import { SearchModal } from '../SearchModal';
```

Comment:
p3;

```suggestion
import { SearchModal } from '@/components/SearchModal';
```

현재 파일 경로와 같은 경로의 파일이 아닌 경우 path alias를 사용합시다.
[Path Alias or Module Resolution in typescript](https://www.typescriptlang.org/docs/handbook/module-resolution.html)